.circleci/config.yml

```diff
@@ -399,20 +414,21 @@ jobs:
         - attach_workspace:
                   at: .
         - checkout
+        - restore-kfp-cache
         - run:
               name: Install system deps
               command: sudo apt update && sudo apt-get install -y libsndfile1 ffmpeg
         - run:
               name: Use py37
               command: |
-                  pyenv install 3.7.0
+                  pyenv install -s 3.7.0
```

Comment:
Why are we using 3.7.0, BTW? "If it works on it, will most likely work on more recent releases"?

Reply:
yeah, i wasnt sure so i left it alone,  will let @andrewtruong comment

Reply:
Nope, I just set this to simplify testing.  [Adding back tests on py36-39 here](https://github.com/wandb/client/pull/3309)